src/payment/strategies/digitalriver/digitalriver-script-loader.ts

```diff
@@ -21,6 +21,6 @@ export default class DigitalRiverScriptLoader {
             throw new PaymentMethodClientUnavailableError();
         }
 
-        return new this._window.DigitalRiver(digitalRiverPublicApiKey, { locale });
+        return Promise.resolve(new this._window.DigitalRiver(publicKey, { locale } ));
```

Comment:
this is an async method. You don't need to wrap the returned type in a Promise.resolve()